src/classes/mainSyncProcess.ts

```diff
@@ -23,7 +23,9 @@ export class MainSyncProcess {
 			FROM TextEntries 
 			WHERE TextTypeId = 67
 			AND TextEntry IS NOT NULL
-			AND TextEntry != ''`;
+			AND TextEntry != ''
+			ORDER BY ID ASC
+			`;
```

Comment:
This just helps us make sure we're always processing the same IDs in the same order